apps/desktop/src/lib/vbranches/branchController.ts

```diff
@@ -226,10 +226,11 @@ export class BranchController {
 		}
 	}
 
-	async unapplyFiles(files: LocalFile[]) {
+	async unapplyFiles(branchId: string, files: LocalFile[]) {
 		try {
 			await invoke<void>('reset_files', {
 				projectId: this.projectId,
+				branchId,
 				files: files.flatMap((f) => f.path).join('\n')
```

Comment:
Can't this be a list of paths right away? My edits now assume this works, and I hope you can validate that.